### PR TITLE
Better docs

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/references/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/references _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: assets/schema_input.json [hunk 1]
```diff
@@ -15,18 +15,21 @@
...
                 "type": "string",
+                "pattern": "^\\S+$",
                 "errorMessage": "Source of genome must be provided and cannot contain spaces",
                 "meta": ["source"]
             },
             "source_vcf": {
                 "type": "string",
-                "errorMessage": "Source of the VCF file must be provided and cannot contain spaces",
+                "pattern": "^\\S+$",
+                "errorMessage": "Source of the VCF, cannot contain spaces",
```
Comment (FriederikeHanssen) [inline assets/schema_input.json:25]: not blocking here. but does source here mean something else than in source_version?

File: assets/schema_input.json [hunk 2]
```diff
@@ -43,78 +46,78 @@
             "bowtie1_index": {
                 "meta": ["bowtie1_index"],
                 "type": "string",
-                "errorMessage": "TODO"
+                "errorMessage": "Bowtie1 index"
```
Comment (FriederikeHanssen) [inline assets/schema_input.json:49]: no pattern? How about unifying the errorMessages? `, cannot contain spaces`

File: assets/schema_input.json (hunk 1 above)
Comment (maxulysse) [reply assets/schema_input.json:25]: yeah, I need better identifier for the fields, I'll fix it in a separate PR

File: assets/schema_input.json (hunk 2 above)
Comment (maxulysse) [reply assets/schema_input.json:49]: it's a folder, so no pattern, but I can add a pattern for no spaces indeed

File: assets/schema_input.json (hunk 1 above)
Comment (maxulysse) [reply assets/schema_input.json:25]: https://github.com/nf-core/references/issues/65

**Pull request merged**